recipes/openmpi/all/conandata.yml

```diff
@@ -0,0 +1,21 @@
+sources:
+  "4.0.4":
+    url: "https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.4.tar.gz"
+    sha1: "e3145347861243d7b0e3876cd196a067609c60be"
+    md5: "48890b54bd0b025918e84f0e47b583b0"
```

Comment:
md5 is not required. sha1 hash is sufficient.

Reply:
Actually, please change this to a `sha256` checksum

Reply:
Also do you need all those versions yourself? If yes, it is fine, if not please only add the latest version and remove the rest